com.webex.Webex.yaml

```diff
@@ -0,0 +1,75 @@
+app-id: com.webex.Webex
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: webex
+separate-locales: false
+tags:
+  - proprietary
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --filesystem=xdg-run/pipewire-0
+  - --env=QT_QPA_PLATFORM=
+  - --own-name=org.kde.*
+  - --talk-name=org.gnome.Shell
+  # Are any of the args below needed?
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --system-talk-name=org.bluez
+  - --system-talk-name=org.freedesktop.login1
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.gtk.Notifications
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=org.freedesktop.secrets
+modules:
+  - shared-modules/libsecret/libsecret.json
+  - name: zypak
+    sources:
+    - type: git
+      url: https://github.com/refi64/zypak
+      branch: webex
```

Comment:
you need to use a revision and not a branch.

Reply:
It has to use the branch right now as the necessary changes are unreleased. We are still trying to find a config that works and then we can refine to match flathub's policies. It is good to know a revision is required now though. I asked the zypak maintainer about merging the branch in a previous comment.

Reply:
changes are in a branch? put the commit.

Reply:
Ah, I see what you mean. I made the change.